runhouse/resources/hardware/utils.py

```diff
@@ -410,16 +410,13 @@ def parse_filters(since: str, cluster_status: str):
 def get_clusters_from_den(cluster_filters: dict):
     get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
 
-    # send den request with filters if the user specifies filters.
     # If "all" filter is specified - get all clusters (no filters are added to get_clusters_params)
     if cluster_filters and "all" not in cluster_filters.keys():
         get_clusters_params.update(cluster_filters)
 
-    # If not filters are specified, get only running clusters.
+    # If no filters are specified get all running clusters.
     elif not cluster_filters:
-        get_clusters_params.update(
-            {"cluster_status": "running", "since": LAST_ACTIVE_AT_TIMEFRAME}
-        )
+        get_clusters_params.update({"cluster_status": "running"})
```

Comment:
@carolineechen you think it makes more sense for us to return all active clusters by default, instead of just the last 24 hours?

Reply:
I think leaning a bit towards option 3 below but tbh not too opinionated as long as the message is clear about what we're displaying. 
1) all running clusters - pro: could be nice to see everything running and maybe you realize you forgot to terminate an old cluster or something. con: if an org has too many running clusters and the list is too long.
2) all running clusters with activity in the last 24 hours - I think this is fine, the message will just need to be clear and explicit otherwise the behavior may seem a little confusing.
3) all running clusters but cap it at most recent 50? 100? - something in between the above two, nice to see all running clusters if there's only a few anyways, but also have a constraint by default. and if all running > the cap, provide a note like `to see all running clusters, use "runhouse cluster list --status running"` (not sure if the 24h is arbitrary or if it's worth also providing "runhouse cluster list --since 24h")